<launch>

    <node name="mission_interpreter" pkg="BathyBoatNav" type="mission_interpreter.py" output="screen">
        <param name="name_mission" type="string" value="test.json"/>
    </node>

</launch>
